bpf/tests/ipsec_from_lxc_generic.h

```diff
@@ -123,11 +123,14 @@ SETUP("tc", "02_ipv4_from_lxc_encrypt")
 int ipv4_from_lxc_encrypt_setup(struct __ctx_buff *ctx)
 {
 	struct node_key node_ip = {};
-	__u32 node_id = NODE_ID;
+	struct node_value node_value = {
+		.id = NODE_ID,
+		.spi = 0,
+	};
 
 	node_ip.family = ENDPOINT_KEY_IPV4;
 	node_ip.ip4 = v4_node_two;
-	map_update_elem(&NODE_MAP, &node_ip, &node_id, BPF_ANY);
+	map_update_elem(&NODE_MAP_V2, &node_ip, &node_value, BPF_ANY);
```

Comment:
Do these map updates need to be mirrored as well?

Reply:
We don't update the NodeMap during cilium runtime from the datapath, so its okay to not have any mirroring here. The datapath won't care much about MapV1's existence.